catalog/transformer/catalog-transformer-streaming-lib/src/test/java/org/codice/ddf/transformer/xml/streaming/lib/TestGenericXmlLib.java

```diff
@@ -119,6 +120,26 @@ public void testNullInputTransform() throws FileNotFoundException, CatalogTransf
     }
 
     @Test
+    public void testNoConfigAndNoFactoriesTransform()
+            throws IOException, CatalogTransformerException {
+        SaxEventHandlerFactory saxEventHandlerFactory = mock(SaxEventHandlerFactory.class);
+        when(saxEventHandlerFactory.getId()).thenReturn("test");
+        SaxEventHandler handler = getNewHandler();
+        when(saxEventHandlerFactory.getNewSaxEventHandler()).thenReturn(handler);
+        XmlInputTransformer xmlInputTransformer = new XmlInputTransformer();
+        xmlInputTransformer.setSaxEventHandlerFactories(Collections.singletonList(
+                saxEventHandlerFactory));
+        MetacardType metacardType = xmlInputTransformer.getMetacardType();
+        assertThat(metacardType.getAttributeDescriptors(),
```

Comment:
I like this more as two separate tests. Would be clearer what regressed/failed. Was it no configuration or no factories?